prime-router/src/main/kotlin/azure/BatchDeciderFunction.kt

```diff
@@ -28,70 +28,66 @@ class BatchDeciderFunction(private val workflowEngine: WorkflowEngine = Workflow
         context: ExecutionContext?,
     ) {
         logger.trace("$batchDecider: Starting")
-        try {
-            workflowEngine.db.transact { txn ->
-                // TODO: if for some reason the batch decider misses proper calculation of a receiver's batch run
-                //  we would want to pull all receivers with BATCH records that have next_action_at in the past and
-                //  merge those with the batchInPrevious60Seconds. Testing shows this to be an unlikely scenario, but
-                //  there is always the chance that something odd happens with the timing of Azure timed functions.
+        workflowEngine.db.transact { txn ->
```

Comment:
Note the only change here is removing the try catch that is swallowing all exceptions